app/Http/Requests/StoreAlbum.php

```diff
@@ -24,7 +24,16 @@ public function authorize()
     public function rules()
     {
         return [
-            //
+            'artists_id' => 'required|integer|exists:artists,id',
+            'title' => 'required|string|max:255',
+            'alt_title' => 'string|max:255',
+            'year' => 'required|integer',
+            'description' => 'max:255',
+            'has_explicit_lyrics' => 'required|boolean',
+            'full_album_price' => 'between:0.00,999.99',
+            'rank' => 'integer|between:1,20',
```

Comment:
We can remove "rank" here because it isn't submitted via request. The value represents the album's overall sales rank, and is computed internally. In fact, it should probably be set as guarded in the model.

Reply:
Yep you are correct, I literally moved my way through the migrations and probably just got ahead of myself.